maplibre/src/context.rs

```diff
@@ -12,11 +12,19 @@ pub struct ViewState {
 }
 
 impl ViewState {
-    pub fn new(window_size: &WindowSize) -> Self {
+    pub fn new(window_size: &WindowSize, style: &Style) -> Self {
```

Comment:
I would favor just passing the `zoom` and `center` manually.

Reply:
I have separated them but I also had to add pitch in the parameters. Should pitch have its own struct too?

Reply:
Looks fine to me. In some other PR I will change the type of the `pitch` though. It should not be f64 but something that can be turned into a `Rad<f64`.